circle.yml

```diff
@@ -18,7 +18,8 @@ deployment:
     branch: master
     commands:
       - make build-all VERSION=head
-      - make publish
+      - make publish-latest
```

Comment:
i don't think we want to update latest when we merge to master, we just want to publish HEAD